web/src/app/pages/edit-survey/edit-job/edit-job.component.ts

```diff
@@ -95,6 +100,19 @@ export class EditJobComponent {
         )
       )
     );
+
+    this.lois = await firstValueFrom(
```

Comment:
Can you add a test to ensure that the correct lois are getting filtered and passed into the loi-selection component?